src/uniref/components/entry/Entry.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { FC } from 'react';
+import { useDispatch } from 'react-redux';
+import { Switch, Route, useRouteMatch } from 'react-router-dom';
+import {
+  InPageNav,
+  Loader,
+  DisplayMenu,
+  TremblIcon,
+  // DropdownButton,
+} from 'franklin-sites';
+
+import {
+  MessageLevel,
+  MessageFormat,
+  MessageType,
+  MessageTag,
+} from '../../../messages/types/messagesTypes';
+
+import EntryMain from './EntryMain';
+
+import BlastButton from '../../../shared/components/action-buttons/Blast';
+import AlignButton from '../../../shared/components/action-buttons/Align';
+import AddToBasketButton from '../../../shared/components/action-buttons/AddToBasket';
+import SideBarLayout from '../../../shared/components/layouts/SideBarLayout';
+import ErrorHandler from '../../../shared/components/error-pages/ErrorHandler';
+
+import UniRefEntryConfig from '../../config/UniRefEntryConfig';
+
+import { addMessage } from '../../../messages/state/messagesActions';
+
+// check if we need it here too and move it to a shared folder if needed
+import { hasContent } from '../../../shared/utils/utils';
+import apiUrls from '../../../shared/config/apiUrls';
+import { LocationToPath, Location } from '../../../app/config/urls';
+
+import useDataApi from '../../../shared/hooks/useDataApi';
+
+import uniRefConverter, {
+  UniRefAPIModel,
+} from '../../adapters/uniRefConverter';
+
+const Entry: FC = () => {
+  const dispatch = useDispatch();
+  const match = useRouteMatch<{ accession: string }>(
+    LocationToPath[Location.UniRefEntry]
+  );
+
+  const { loading, data, status, error, redirectedTo } = useDataApi<
+    UniRefAPIModel
+  >(apiUrls.uniref.entry(match?.params.accession || ''));
+
+  if (error || !match?.params.accession) {
+    return <ErrorHandler status={status} />;
+  }
+
+  if (loading || !data) {
+    return <Loader />;
+  }
+
+  if (redirectedTo) {
+    const message: MessageType = {
+      id: 'job-id',
+      content: `You are seeing the results from: ${redirectedTo}.`,
+      format: MessageFormat.IN_PAGE,
+      level: MessageLevel.SUCCESS,
+      dateActive: Date.now(),
+      dateExpired: Date.now(),
+      tag: MessageTag.REDIRECT,
+    };
+
+    dispatch(addMessage(message));
+  }
+
+  const transformedData = uniRefConverter(data);
+
+  const sections = UniRefEntryConfig.map((section) => ({
+    label: section.name,
+    id: section.name,
+
+    disabled: !hasContent(transformedData[section.name]),
+  }));
+
+  const displayMenuData = [
+    {
+      name: 'Entry',
```

Comment:
As we don't have "tabs", only the in-page nav is needed, not the "Entry" header - UX chat this morning also points to this being moved to actual tabs in the future, so up to you if you want to keep it in or remove it now.